dns/bind/src/opnsense/mvc/app/controllers/OPNsense/Bind/Api/RecordController.php

```diff
@@ -52,6 +53,9 @@ private function setDomainSerial($uuid)
             if ($record !== null) {
                 (new Domain)->updateSerial((string)$record->domain)->serializeToConfig();
                 Config::getInstance()->save();
+                $backend = new Backend();
```

Comment:
This function was written by @AdSchellevis .. for me it's ok, but I'd like to have him a quick look

Reply:
If this is hit per record, you really don't want to do this. Usually we inform the user that an apply is needed.

Reply:
I'm not sure, If I add an record I want it to be available. You wouldn't need the serial to be updated otherwise.
If you want the User to be Informed, I realy would appreciate some help. I'm not that firm with JavaScript/Php/GUI

Reply:
It only updated in the config.xml, at the end you have to hit apply to reload the template and restart the daemon. Maybe all you need for your ongoing project is to call api call fro service reconfigure?